openvpn/anonine/Sweden.ovpn

```diff
@@ -1,19 +1,26 @@
 client
-dev tun0
-proto udp 
+dev tun
+proto tcp
 ; Cert
-ns-cert-type server
-cipher BF-CBC
+remote-cert-tls server
+cipher AES-256-CBC
 ;Host
 resolv-retry infinite
 ;auth
-auth-user-pass /config/openvpn-credentials.txt
+auth-user-pass
+auth-nocache
+keepalive 10 30
```

Comment:
Keepalive will not work. Or. We're recommending running with ping-exit that makes openvpn exit when there are connection issues. That way docker can restart the container when something hangs or fails.